.changeset/happy-singers-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+"openapi-fetch": minor
```

Comment:
Just a note for next time: we’re using `minor` patches only for breaking changes while we’re on `0.x` on `openapi-fetch`. `patch` includes feature additions and bugfixes before `1.0`.